test/bzmesh.jl

```diff
@@ -7,16 +7,16 @@ include("testcases.jl")
 
 
 @testset "bzmesh_uniform agrees with spglib" begin
-    spglib = pyimport_conda("spglib", "spglib")
 
     function test_against_spglib(kgrid_size; kshift=[0, 0, 0])
         kgrid_size = Vec3(kgrid_size)
-        identity = [reshape(Mat3{Int}(I), 1, 3, 3)]
+        identity = collect(reshape(Matrix{Cint}(I, 3, 3), 1, 3, 3))
```

Comment:
I feel a bit uneasy that c types appear here. Maybe you should add a higher-level way to call this function without C types?